src/ephemeris.c

```diff
@@ -257,7 +254,65 @@ u8 ephemeris_good(const ephemeris_t *eph, gps_time_t t)
 
   /* TODO: this doesn't exclude ephemerides older than a week so could be made
    * better. */
-  return (eph->valid && eph->healthy && fabs(dt) < EPHEMERIS_VALID_TIME);
+  return (eph->valid && fabs(dt) < ((u32)eph->fit_interval)*60*60);
+}
+
+/** Convert a GPS URA index into a value.
+*
+* |param index URA index.
+* |return the URA in meters.
+*/
+float decode_ura_index(const u8 index) {
+  static float values[16] = {
+    2.0f,    // 0
```

Comment:
Consider using c99 style: `[0] = 2.0f,`, or just removing the comments showing the array indices.